views/make/index.js

```diff
@@ -73,6 +73,13 @@ module.exports = view.extend({
         goPreviousView: function (e) {
             global.history.back();
         },
+        onBackClick: function () {
+            if (this.mode !== 'edit') {
+                this.goPreviousView();
+            } else {
+                this.page('/profile');
```

Comment:
That's actually a whole can of worms because "back in history" can mean back to sub views of the editor (like Add A Brick), which means you can never get out.